apps/judicial-system/web/src/routes/Shared/Cases/MobileCase.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+
+import { Case, isInvestigationCase } from '@island.is/judicial-system/types'
+import {
+  capitalize,
+  caseTypes,
+  formatDOB,
+} from '@island.is/judicial-system/formatters'
+import { Box, Text, FocusableBox, Tag } from '@island.is/island-ui/core'
+
+import * as styles from './MobileCase.css'
+import { mapCaseStateToTagVariant } from './utils'
+
+interface CategoryCardProps {
+  heading: string | React.ReactNode
+  tags?: React.ReactNode
+  onClick: () => void
+}
+
+const CategoryCard: React.FC<CategoryCardProps> = ({
```

Comment:
CategoryCard in island-ui was written with a single use case in mind. So I decided to recreate it.